packages/manager/apps/web-ongoing-operations/src/components/Modal/Content/MeDocument.component.tsx

```diff
@@ -22,7 +22,13 @@ export default function MeDocumentComponent({
 
   return (
     <OdsLink
-      label={t(`domain_operations_update_nicowner_click_${argumentKey}`)}
+      label={t(
+        `domain_operations_update_nicowner_click_${
+          !argumentKey || argumentKey !== 'corporationProof'
```

Comment:
`argumentKey === "corporationProof ? 'nicowner' : argumentKey ?? ""`

I prefer this syntax, but it's not mandatory to use it